src/java/org/apache/cassandra/config/CassandraRelevantProperties.java

```diff
@@ -453,7 +453,7 @@ public enum CassandraRelevantProperties
     SAI_SORTED_TERMS_PARTITION_BLOCK_SHIFT("cassandra.sai.sorted_terms_partition_block_shift", "4"),
 
     SAI_TEST_BALANCED_TREE_DEBUG_ENABLED("cassandra.sai.test.balanced_tree_debug_enabled", "false"),
-    SAI_TEST_DISABLE_TIMEOUT("cassandra.sai.test.disable.timeout", "false"),
+    SAI_TEST_DISABLE_TIMEOUT("cassandra.sai.test.timeout_disabled", "false"),
```

Comment:
I have changed this mainly because it didn't fall into the same naming format as other, similar, properties

Reply:
I guess the only other option would be to use "enabled" rather than "disabled" verbiage and flip the default.

Reply:
I thought about but I don't like the idea of having the default as `true`. Not sure why apart from this is a debug / test setting so I prefer it be explicit what the flag is doing.